File: .NET Word Examples/MergeApiDemo.cs
using Spire.Cloud.Word.Sdk.Api;
using Spire.Cloud.Word.Sdk.Client;
using Spire.Cloud.Word.Sdk.Model;
using System;
using System.Collections.Generic;

namespace SpireDocCloudDemo
{
    class MergeApiDemo
    {
        static string appId = "your id";
        static string appKey = "your key";

        public static void MergeDocument()
        {
            Configuration wordConfiguration = new Configuration(appId, appKey);
            MergeApi mergeApi = new MergeApi(wordConfiguration);
            string name = "mergeDocument1.docx";
            List<MergingFile> files = new List<MergingFile>() {
                new MergingFile("mergeDocument2.docx", "input", null)
            };
            string folder = "input";
            string storage = null;
            string password = null;
            string destFilePath = "output/mergeDocument_output.docx";
            mergeApi.MergeDocument(name, files, folder, storage, password, destFilePath);
        }
    }
}

File: Java Word Examples/MergeApiDemo.java
import spire.cloud.word.sdk.client.*;
import spire.cloud.word.sdk.client.api.MergeApi;
import spire.cloud.word.sdk.client.model.MergingFile;

import java.util.*;

public class MergeApiDemo {
    static String appId = "your id";
    static String appKey = "your key";
    static String baseUrl = "https://api.e-iceblue.cn";
    static Configuration wordConfiguration = new Configuration(appId, appKey, baseUrl);
    static MergeApi mergeApi = new MergeApi(wordConfiguration);

    public static void mergeDocument() throws ApiException {
        String name = "mergeDocument1.docx";
        List<MergingFile> files = new ArrayList<MergingFile>();
        files.add(new MergingFile("input/mergeDocument2.docx",null,null));
        String folder = "input";
        String storage = null;
        String password = null;
        String destFilePath = "output/mergeDocument_output.docx";
        mergeApi.mergeDocument(name, files, destFilePath, folder, storage, password);
    }
}
